lib/agent0/agent0/hyperdrive/crash_report/crash_report.py

```diff
@@ -109,14 +105,25 @@ def build_crash_trade_result(
     ---------
     exception : Exception
         The exception that was thrown
+    agent : HyperdriveAgent
+        Object containing a wallet address and Elfpy Agent for determining trades
+    trade_object : types.Trade[HyperdriveMarketAction]
+        A trade provided by a HyperdriveAgent
+    hyperdrive : HyperdriveInterface
+        An interface for Hyperdrive with contracts deployed on any chain with an RPC url.
+
+    Returns
+    -------
+    TradeResult
     """
     trade_result = TradeResult(
         status=TradeStatus.FAIL,
         agent=agent,
         trade_object=trade_object,
     )
+    pool_state = hyperdrive.current_pool_state
```

Comment:
Crash reporting is very dependent on getting data from the block in the exception if it's provided. This functionality has been removed with these changes.